Makefile

```diff
@@ -206,43 +206,43 @@ endif
 ZMQ_CHECK = $(DEP_INSTALL_DIR)/checkZMQ.chpl
 check-zmq: $(ZMQ_CHECK)
 	@echo "Checking for ZMQ"
-	$(CHPL) $(CHPL_FLAGS) $(ARKOUDA_COMPAT_MODULES) $< -o $(DEP_INSTALL_DIR)/$@
+	@$(CHPL) $(CHPL_FLAGS) $(ARKOUDA_COMPAT_MODULES) $< -o $(DEP_INSTALL_DIR)/$@ && ([ $$? -eq 0 ] && echo "Success compiling program") || echo "\nERROR: Please ensure that dependencies have been installed correctly (see -> https://github.com/Bears-R-Us/arkouda/blob/master/pydoc/setup/BUILD.md)\n"
```

Comment:
Thanks for taking this on, @bmcdonald3 !

I'm not at all a bash expert, but I probably would've just written this `|| echo "\nERROR: Please ensure that dependencies have been installed correctly (see -> https://github.com/Bears-R-Us/arkouda/blob/master/pydoc/setup/BUILD.md)\n"`  Specifically, I don't think a success message is necessary and the `[ $$? -eq 0]` feels redundant with what the && / || are doing inherently (?).

Reply:
I do agree with @bradcray here, I do not know that we need the `Success compiling program` message. However, I do not believe that it will hurt anything if we decide to keep it.